lmfdb/elliptic_curves/elliptic_curve.py

```diff
@@ -717,20 +720,40 @@ def __init__(self):
             label="Semistable",
             example="Yes",
             knowl="ec.semistable")
+        cm_opts = []
+        for t in cm_j_invariants_and_orders(QQ):
+            el = (str(t[0]*t[1]**2), t[0]*t[1]**2)
+            if not el in cm_opts:
+                cm_opts.append(el)
+        cm_opts = sorted(cm_opts, key =lambda el: el[1], reverse = True)
+        cm_opts.insert(0, ("",""))
+        #cm_opts = [('', ''), (-3, -3), (-4, -4), (-7, -7), (-8, -8), (-11, -11), (-12, -12), (-16, -16), (-19, -19),
+                #(-27, -27), (-28, -28), (-43, -43), (-67, -67), (-163, -163)]
```

Comment:
If you want to leave this in as a comment I would just change it to a the list of CM options (rather than pairs), and include the ones you were missing.